.github/workflows/CD.yml

```diff
@@ -29,7 +30,24 @@ jobs:
         with:
           java-version: ${{ matrix.java }}
 
-      - name: Run build
+      - name: Build library
```

Comment:
Nit: Should we call it "Build rpm deb Artifacts"?

Reply:
You are right. Because they also send artifacts to S3, I changed name to Build and ship library artifacts